### DISCUSSION TITLE
How can I use Gravatar as my profile picture?

### DISCUSSION THREAD

**Discussion by christianspecht** [community/community#38110] [Other Feature Feedback, Questions, & Ideas] [answered]
I've had a Gravatar account for >10 years, and it's linked to the one and only email adress in my GitHub settings.

I always *thought* Github would use my Gravatar, but I just noticed that the image on my GitHub profile is actually coming from `https://avatars.githubusercontent.com/u/263372?v=4`

[On this page in the Github docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#changing-your-profile-picture), there's the following image which implies that I can **either** upload an avatar **or** use my Gravatar:

![upload a photo](https://docs.github.com/assets/cb-108208/images/help/profile/edit-profile-picture-options.png)

But when I try to edit my profile picture, the "Revert to Gravatar" option is not there:
![my options](https://user-images.githubusercontent.com/263372/199607745-072cef35-b396-4519-904f-b8d2afed7ed2.png)

**So how can I make Github use my Gravatar, for the email adress that's already associated with GitHub?**

**Comment by radar-blip** [comment]
I don't know if this will help at all, but if you read the selected answer here #32845, 
maybe you can make some kind of sense out of what might be happening.

**Comment by christianspecht** [comment, marked as answer]
Apparently those `https://avatars.githubusercontent.com/u/263372?v=4` URLs kind of "cache" Gravatar or redirect to it.

I changed my image on Gravatar yesterday (for the email that's linked with GitHub), and my GitHub profile now shows my new Gravatar image...but the URL is still `https://avatars.githubusercontent.com/u/263372?v=4`.

That explains why I'm not seeing the "Revert to Gravatar" option...but it's not obvious that GitHub actually uses my Gravatar, because of the different URL.

**Reply by uzkjzjjz** [reply]
@zackfern

**Comment by dgchrt** [comment]
I've been through the same situation and in my case I can change my Gravatar picure all I want, I still get the same old picture on GitHub. No idea how to fix this.

**Comment by nithin-mk** [comment]
@dgchrt This may fix your issue: https://webapps.stackexchange.com/a/28160/372743
